allocator.go

```diff
@@ -8,51 +8,18 @@ import (
 	"time"
 )
 
-// nodeId is a 64-bit identifier for nodes.
-type nodeId int64
-
-// rangeId is a 64-bit identifier for ranges.
-type rangeId int64
-
-// resource is 32-bit identifier for resources -> {disk, qps}
-type resource int
-
 // Allocation is the return type of our allocator.
-// It models mappings of RangeIDs to a list of NodeIDs.
```

Comment:
Changed all **Ranges** to **Replicas**. 

Because `range `is a key word in GoLang and things will be cleaner if we avoid having to deal with it.